app-backend/api/src/main/scala/tasks/Routes.scala

```diff
@@ -649,4 +655,30 @@ trait TaskRoutes
         }
       }
     }
+
+  def getActiveTaskSession(taskId: UUID): Route =
+    authenticate { user =>
+      authorizeScope(
+        ScopedAction(Domain.AnnotationProjects, Action.ReadTasks, None),
+        user
+      ) {
+        authorizeAsync {
+          TaskSessionDao
+            .authorized(
+              taskId,
+              user,
+              ActionType.View
+            )
+            .transact(xa)
+            .unsafeToFuture
+        } {
+          complete {
+            TaskSessionDao
+              .getActiveSessionByTaskId(taskId, user)
+              .transact(xa)
+              .unsafeToFuture
+          }
```

Comment:
I'm a little confused about the response when there's no active session. If I'm a client and I try to get a single item and you're the server and can't find one, I think I'd be surprised that you sent me a 200 with no body. I'd expect a 204 if you're telling me "I don't have anything for you, but it's fine"